aws/sra-test/integration-tests/aws-sdk-s3/tests/request_information_headers.rs

```diff
@@ -31,228 +128,267 @@ struct FixupPlugin {
 // # Client waits 1 second between retry attempts.
 #[tokio::test]
 async fn three_retries_and_then_success() {
```

Comment:
It should be possible to add this directly to the S3 integration tests now, although the S3 crate doesn't compile in orchestrator mode right now due to presigning still directly referencing middleware.